yt_dlp/compat/urllib/__init__.py

```diff
@@ -0,0 +1,7 @@
+# flake8: noqa: F405
+from urllib import *  # noqa: F403
+
```

Comment:
```suggestion
```
Do we really need to reimport here?

Reply:
this is what pukkandan suggested afaik and is what is standard in the compat

mainly just need the import for get_proxies here instead though

Reply:
Why are we passing through the whole module anyways? Wouldn't it be easier to just create a `compat_get_proxies` method instead, since thats the only use of the function?

I guess that would also be conflicting with the other style...

Reply:
@pukkandan since you suggested this

Reply:
The idea is that we don't need to touch the code to add/remove the compat function, only imports (See my change). It's not very applicable for this case, but very useful for functions used in extractors like functools/re. So we might as well conserve the style here.

I'll look into removing the * import separately. For now leave it be.